src/api/app/controllers/application_controller.rb

```diff
@@ -89,6 +89,13 @@ def self.validate_action(opt)
 
   protected
 
+  def feature_enabled?(feature)
+    return if Flipper.enabled?(feature.to_sym, User.possibly_nobody) || Flipper.enabled?(feature.to_sym)
```

Comment:
Why `User.possibly_nobody`? IMO we should user `User.session` that is not taking into account `_nobody_` user.

Reply:
done!